litgpt/__main__.py

```diff
@@ -1,5 +1,26 @@
 # Copyright Lightning AI. Licensed under the Apache License 2.0, see LICENSE file.
 
+import warnings
+import re
+# There is currently a PyTorch bug that raises a false-positive warning.
+# For more information, see https://github.com/Lightning-AI/litgpt/issues/1561
+warning_message = (
```

Comment:
It's not necessary to have the entire warning message spelled out here. You should be able to just use
```py
"The epoch parameter in `scheduler.step()` was not necessary and is being deprecated.*"
```